python/pyspark/sql/tests/arrow/test_arrow_python_udf.py

```diff
@@ -244,6 +244,39 @@ def test_udf_use_arrow_and_session_conf(self):
             )
 
 
+@unittest.skipIf(
+    not have_pandas or not have_pyarrow, pandas_requirement_message or pyarrow_requirement_message
+)
+class ArrowPythonUDFLegacyTestsMixin(ArrowPythonUDFTestsMixin):
```

Comment:
do we need to also change `pyspark.sql.tests.connect.arrow.test_parity_arrow_python_udf`?